lib/KTable/KTable.vue

```diff
@@ -0,0 +1,234 @@
+<template>
+
+  <div class="k-table">
+    <table>
+      <caption v-if="caption">
+        {{ caption }}
+      </caption>
+      <thead>
+        <tr>
+          <th
+            v-for="(header, index) in headers"
+            :key="index"
+            tabindex="0"
+            @click="handleSort(index)"
+            @keydown="handleKeydown($event, -1, index)"
+          >
+            {{ header.label }}
+            <span v-if="header.dataType !== 'others'" class="sort-icon">
+              <span v-if="sortKey === index && sortOrder === 'asc'">⬆️</span>
```

Comment:
In the next iteration, could be nice to consider using icons as illustrated in the design
![table](https://github.com/learningequality/kolibri-design-system/assets/13509191/36fe3207-be1a-4e0e-bc80-77801d8c1a93)